six/modules/c++/six.sidd/source/DerivedXMLParser200.cpp

```diff
@@ -806,14 +820,32 @@ void DerivedXMLParser200::parseColorSpaceTransformFromXML(
 {
     XMLElem manageElem = getFirstAndOnly(colorElem, "ColorManagementModule");
 
-    parseEnum(getFirstAndOnly(manageElem, "RenderingIntent"),
-            transform.colorManagementModule.renderingIntent);
+    std::string renderIntentStr;
```

Comment:
Had to do this work around because fixing wasn't possible. Either because of invisible chars or #defines with the same name somewhere else. Wasn't able to figure out why.

Reply:
works for me

Reply:
(and by works for me i mean the workaround make sense to me)